plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,268 @@
+    * restore the DBs on remote servers
+    * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
+
+## Prepare remote Postgres
+GOAL: To use remote Postgres database with Katello we have to:
+* be able to access the databases from katello box
+* the database user we use to connect to the database needs to own the database, i.e. it can create, alter and delete the tables, indexes and constraints. Note it is not required to be able to create the database itself.
+
+### Install Postgres
+Warning: This is just minimal testing setup which is not suitable for production.
+
+Assume our postgres server has hostname `postgres.example.com`.
+
+```
+yum install -y postgresql-server postgresql-contrib
+postgresql-setup initdb
+systemctl enable --now postgresql
+```
+
+Now we need to make Postgres listen to inbound connections, please adjust these parameters to your own networking and security requirements. 
+
+Edit `/var/lib/pgsql/data/postgresql.conf`. Uncomment `listen_address` and modify its value to look like: 
+```
+  listen_address = "*"
+```
+
+The next step we need to take is to add a proper client authentication for remote client to our postgres server. To achieve the same, edit `/var/lib/pgsql/data/pg_hba.conf`.
+Append the following line at the end of the file
+```
+  host          all          all          <katello.example.com ip>/24              md5
+```
+
+Now restart the postgres service for changes to take effect
+```
+systemctl restart postgresql
+```
+
+### Create the databases
+Switch the user role to postgres and start postgres client
+```
+su - postgres -c psql
+```
+
+Once inside the client, we need to create two databases and dedicated roles, one for foreman and one for candlepin
+```sql
+CREATE USER "foreman" WITH PASSWORD '<FOREMAN_PASSWORD>';
+CREATE USER "candlepin" WITH PASSWORD '<CANDLEPIN_PASSWORD>';
+CREATE DATABASE foreman OWNER foreman;
+CREATE DATABASE candlepin OWNER candlepin;
+```
+
+### Test it works
+From `katello.example.com` test the DB is accessible:
+```
+PGPASSWORD='<FOREMAN_PASSWORD>' psql -h postgres.example.com  -p 5432 -U foreman -d foreman -c "SELECT 1 as ping"
+```
+```
+PGPASSWORD='<CANDLEPIN_PASSWORD>' psql -h postgres.example.com  -p 5432 -U candlepin -d candlepin -c "SELECT 1 as ping"
+```
+If there are no errors we are done with database preparation. 
+
+## Prepare remote Mongo
+GOAL: To use remote Mongo database with Katello we have to:
+* be able to access the databases from katello box
+* the database user we use to connect to the database needs to own the database
+
+### Install Mongo DB
+Warning: This is just minimal testing setup which is not suitable for production.
+
+Assume our Mongo server has hostname `mongo.example.com`.
+Install and enable Mongo server
+``` 
+yum -y localinstall https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum install -y mongodb mongodb-server
```

Comment:
We should avoid encouraging installation of EPEL MongoDB and use RHSCL examples since that is what we support internally.

Reply:
How does one do that?

Reply:
Install centos-release-scl and then rh-mongodb34-syspaths